README.md

```diff
@@ -29,8 +29,11 @@ Right click anywhere within the file you are currently editing. Your cursor posi
 # Installation
 The easiest is to install using [Package control](https://sublime.wbond.net/).
 
-To install manually, you can clone Git repository directly:
-* `cd ~/Library/Application Support/Sublime Text 2/Packages/`
+To install manually, you can clone the Git repository directly:
+* Mac:
+  * `cd ~/Library/Application Support/Sublime Text 3/Packages/`
```

Comment:
I'm guessing this is still the correct path for Sublime 3, but I don't have a Mac to test it on.

Reply:
For ST4, you can drop the ` 3`. If you have a no-3 Packages directory, ST4 will use it. If not, it will use the 3 one.

Reply:
You mean something like `~/Library/Application Support/Sublime Text/Packages`?

Reply:
Yes. Exactly that.

Reply:
Cool, got that updated.